EventFiltering/PWGHF/HFFilter.cxx

```diff
@@ -752,12 +752,17 @@ struct HfFilter { // Main struct for HF triggers
     auto bc = collision.template bc_as<aod::BCsWithTimestamps>();
     if (computeTPCPostCalib && currentRun != bc.runNumber()) {
 
-      auto calibList = ccdb->getForTimeStamp<TList>(fConfigCcdbPathTPC.value, bc.timestamp());
+      auto calibList = ccdb->getForTimeStamp<TList>(ccdbPathTPC.value, bc.timestamp());
 
-      hMapPionMean = (TH3F*)calibList->FindObject("mean_map_pion");
-      hMapPionSigma = (TH3F*)calibList->FindObject("sigma_map_pion");
-      hMapProtonMean = (TH3F*)calibList->FindObject("mean_map_proton");
-      hMapProtonSigma = (TH3F*)calibList->FindObject("sigma_map_proton");
+      if (!calibList->FindObject("mean_map_pion") || !calibList->FindObject("sigma_map_pion") || !calibList->FindObject("mean_map_proton") || !calibList->FindObject("sigma_map_proton"))
+        LOG(fatal) << "Can not find histograms!";
+      else {
+
+        hMapPionMean = (TH3F*)calibList->FindObject("mean_map_pion");
+        hMapPionSigma = (TH3F*)calibList->FindObject("sigma_map_pion");
+        hMapProtonMean = (TH3F*)calibList->FindObject("mean_map_proton");
+        hMapProtonSigma = (TH3F*)calibList->FindObject("sigma_map_proton");
+      }
```

Comment:
You are loading everything twice there unnecessarily. First load all the histograms and then check the validity of the pointers `hMapPionMean`,...
Also you should check that `calibList` right after loading it.